ci/init.js

```diff
@@ -3,13 +3,15 @@ const tracer = require('../packages/dd-trace')
 const { ORIGIN_KEY } = require('../packages/dd-trace/src/constants')
 const { isTrue } = require('../packages/dd-trace/src/util')
 
+const isJestWorker = () => !!process.env.JEST_WORKER_ID
+
 const options = {
   startupLogs: false,
   tags: {
     [ORIGIN_KEY]: 'ciapp-test'
   },
   isCiVisibility: true,
-  flushInterval: 5000
+  flushInterval: isJestWorker() ? 120000 : 5000
```

Comment:
Sounds good, the interval is mostly useful for long-lived processes that otherwise have no guarantee to send traces before a long time.

As a future improvement, I wonder if we should consider not even have an interval for CI, and instead rely on something like span count or only when the process is going to exit.